lib/cosmos/gui/opengl/gl_viewer.rb

```diff
@@ -532,15 +532,15 @@ def worldToEyeZ(w)
       w[0]*@transform[0][2] + w[1]*@transform[1][2] + w[2]*@transform[2][2] + @transform[3][2]
     end
 
+    def calc_prime(v)
+      return [v[0]*@itransform[0][0] + v[1]*@itransform[1][0] + v[2]*@itransform[2][0] + @itransform[3][0],
+              v[0]*@itransform[0][1] + v[1]*@itransform[1][1] + v[2]*@itransform[2][1] + @itransform[3][1],
+              v[0]*@itransform[0][2] + v[1]*@itransform[1][2] + v[2]*@itransform[2][2] + @itransform[3][2]]
+    end
+
```

Comment:
This was the biggest code duplication issue we had so I threw in this change too. I tested it and it still works.